ui/src/test/resources/user_ca_response.json

```diff
@@ -0,0 +1,26 @@
+{
+  "token": "usr-f9154016-94e8-4686-a840-075688ac07b5",
+  "status": "PRE_ACTIVATED",
+  "createdOn": "2017-10-30T22:15:45",
+  "clientUserId": "123456",
+  "profileType": "INDIVIDUAL",
+  "firstName": "Person",
+  "lastName": "FromCanada",
+  "dateOfBirth": "1991-01-01",
+  "email": "user+4satF1xV@hyperwallet.com",
+  "addressLine1": "950 Granville",
+  "city": "Vancouver",
+  "stateProvince": "BC",
+  "country": "CA",
+  "postalCode": "H0H0H0",
+  "language": "en",
+  "programToken": "prg-83836cdf-2ce2-4696-8bc5-f1b86077238c",
+  "links": [
+    {
+      "params": {
+        "rel": "self"
+      },
+      "href": "https://api.sandbox.hyperwallet.com/rest/v3/users/usr-f9154016-94e8-4686-a840-075688ac07b5"
```

Comment:
@peter-joseph Could you change real link :
https://localhost:8181/rest/v3/users/usr-123